vowpalwabbit/core/src/parse_args.cc

```diff
@@ -671,6 +671,7 @@ void parse_feature_tweaks(options_i& options, VW::workspace& all, bool interacti
                .keep()
                .help("Read a dictionary for additional features (arg either 'x:file' or just 'file')"))
       .add(make_option("dictionary_path", dictionary_path)
```

Comment:
This should not be kept as it could be different on each system.

Reply:
can't be overriden, removed